azurecontainerapps/src/ContainerAppHelper.ts

```diff
@@ -159,7 +159,7 @@ export class ContainerAppHelper {
             try {
                 let command = `containerapp ingress update -n ${containerAppName} -g ${resourceGroup}`;
                 if (!util.isNullOrEmpty(ingress)) {
-                    command += ` --ingress ${ingress}`;
+                    command += ` --type ${ingress}`;
```

Comment:
Could you please share more info about this update?
Also, I see that the GHA is using --ingress: https://github.com/Azure/container-apps-deploy-action/blob/02d045129c058b13312276b50a552834cf74e2af/src/ContainerAppHelper.ts#L153
Could you please clarify why we need these to be different?

Reply:
@daniv-msft 
The GHA change you're referencing to is when ingress was set using `up` command. The change here is for `ingress update` command, where instead of taking `--ingress`, it take `--type` to set ingress type to `external` or `internal`.

The update here is mainly for users that already have a container app, and wish to update their container app using this deploy task while also specifying ingress updates. In the past, if the user was using managed identity to pull from ACR for the existing app, it'll fail because up command doesn't work with MI. And in order to fix that, we now use `containerapp update` and `containerapp ingress update` in order to update the container app **AND** set the ingress without requiring users to enable admin credentials.